internal/pkg/container/kernel.go

```diff
@@ -0,0 +1,47 @@
+package container
+
+import (
+	"path"
+	"path/filepath"
+	"sort"
+
+	"github.com/hpcng/warewulf/internal/pkg/wwlog"
+)
+
+var (
+	kernelSearchPaths = []string{
+		// This is a printf format where the %s will be the kernel version
+		"/boot/vmlinuz-*",
+		"/lib/modules/*/vmlinuz*",
+	}
+)
+
+func KernelFind(container string) string {
+	container_path := RootFsDir(container)
+	if container_path == "" {
+		return ""
```

Comment:
Should we consider returning an error or (ok == false) here and below? Does the caller need to check for an empty string returned here and NOT_FOUND below?

Reply:
Yeah, as it stands now, if it isn't found, it returns empty string.

Reply:
Cool - makes sense for the caller to look for one and only thing.